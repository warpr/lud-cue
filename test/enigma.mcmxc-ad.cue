REM COMMENT l√ªd cue v1
REM MUSICBRAINZ_ALBUM_ID c84d5361-750b-3d42-887c-13e315cd85f1
REM LUD_DURATION_IN_SECONDS 2420.824000
CATALOG 077778622420
TITLE "MCMXC a.D."
PERFORMER "Enigma"
FILE "disc1.m4a" WAVE
  TRACK 01 AUDIO
    TITLE "The Voice of Enigma"
    PERFORMER "Enigma"
    ISRC DEG129000051
    INDEX 01 0:00:00
  TRACK 02 AUDIO
    TITLE "Principles of Lust: Sadeness / Find Love / Sadeness (reprise)"
    PERFORMER "Enigma"
    ISRC DEG129000052
    INDEX 01 2:21:60
    INDEX 02 6:30:55
    INDEX 03 11:16:07
  TRACK 03 AUDIO
    TITLE "Callas Went Away"
    PERFORMER "Enigma"
    ISRC DEG129000053
    INDEX 01 14:05:55
  TRACK 04 AUDIO
    TITLE "Mea Culpa"
    PERFORMER "Enigma"
    ISRC DEG129000054
    INDEX 01 18:33:45
  TRACK 05 AUDIO
    TITLE "The Voice & The Snake"
    PERFORMER "Enigma"
    ISRC DEG129000055
    INDEX 01 23:37:02
  TRACK 06 AUDIO
    TITLE "Knocking on Forbidden Doors"
    PERFORMER "Enigma"
    ISRC DEG129000056
    INDEX 01 25:17:00
  TRACK 07 AUDIO
    TITLE "Back to the Rivers of Belief: Way to Eternity / Hallelujah / The Rivers of Belief"
    PERFORMER "Enigma"
    ISRC DEG129000057
    INDEX 01 29:48:37
    INDEX 02 32:09:40
    INDEX 03 36:15:00
