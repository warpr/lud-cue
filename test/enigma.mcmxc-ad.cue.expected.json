{
    "artist": "Enigma",
    "barcode": "077778622420",
    "comments": ["l√ªd cue v1"],
    "duration": 2420.824,
    "files": [
        {
            "filename": "disc1.m4a",
            "format": "WAVE",
            "tracks": [
                {
                    "artist": "Enigma",
                    "duration": 141.8,
                    "pos": 1,
                    "start": {
                        "index": 1,
                        "minutes-seconds-frames": "0:00:00",
                        "seconds": 0
                    },
                    "title": "The Voice of Enigma"
                },
                {
                    "artist": "Enigma",
                    "duration": 248.93333333333334,
                    "pos": 2,
                    "start": {
                        "index": 1,
                        "minutes-seconds-frames": "2:21:60",
                        "seconds": 141.8
                    },
                    "groupTitle": "Principles of Lust",
                    "title": "Sadeness"
                },
                {
                    "artist": "Enigma",
                    "duration": 285.36,
                    "pos": 2,
                    "start": {
                        "index": 2,
                        "minutes-seconds-frames": "6:30:55",
                        "seconds": 390.73333333333335
                    },
                    "groupTitle": "Principles of Lust",
                    "title": "Find Love"
                },
                {
                    "artist": "Enigma",
                    "duration": 169.64,
                    "pos": 2,
                    "start": {
                        "index": 3,
                        "minutes-seconds-frames": "11:16:07",
                        "seconds": 676.0933333333334
                    },
                    "groupTitle": "Principles of Lust",
                    "title": "Sadeness (reprise)"
                },
                {
                    "artist": "Enigma",
                    "duration": 267.86666666666656,
                    "pos": 3,
                    "start": {
                        "index": 1,
                        "minutes-seconds-frames": "14:05:55",
                        "seconds": 845.7333333333333
                    },
                    "title": "Callas Went Away"
                },
                {
                    "artist": "Enigma",
                    "duration": 303.42666666666673,
                    "pos": 4,
                    "start": {
                        "index": 1,
                        "minutes-seconds-frames": "18:33:45",
                        "seconds": 1113.6
                    },
                    "title": "Mea Culpa"
                },
                {
                    "artist": "Enigma",
                    "duration": 99.97333333333336,
                    "pos": 5,
                    "start": {
                        "index": 1,
                        "minutes-seconds-frames": "23:37:02",
                        "seconds": 1417.0266666666666
                    },
                    "title": "The Voice & The Snake"
                },
                {
                    "artist": "Enigma",
                    "duration": 271.49333333333334,
                    "pos": 6,
                    "start": {
                        "index": 1,
                        "minutes-seconds-frames": "25:17:00",
                        "seconds": 1517
                    },
                    "title": "Knocking on Forbidden Doors"
                },
                {
                    "artist": "Enigma",
                    "duration": 141.03999999999996,
                    "pos": 7,
                    "start": {
                        "index": 1,
                        "minutes-seconds-frames": "29:48:37",
                        "seconds": 1788.4933333333333
                    },
                    "groupTitle": "Back to the Rivers of Belief",
                    "title": "Way to Eternity"
                },
                {
                    "artist": "Enigma",
                    "duration": 245.4666666666667,
                    "pos": 7,
                    "start": {
                        "index": 2,
                        "minutes-seconds-frames": "32:09:40",
                        "seconds": 1929.5333333333333
                    },
                    "groupTitle": "Back to the Rivers of Belief",
                    "title": "Hallelujah"
                },
                {
                    "artist": "Enigma",
                    "duration": 245.82400000000007,
                    "pos": 7,
                    "start": {
                        "index": 3,
                        "minutes-seconds-frames": "36:15:00",
                        "seconds": 2175
                    },
                    "groupTitle": "Back to the Rivers of Belief",
                    "title": "The Rivers of Belief"
                }
            ]
        }
    ],
    "mbid": "https://musicbrainz.org/release/c84d5361-750b-3d42-887c-13e315cd85f1#_",
    "title": "MCMXC a.D."
}
